samples/drivers/mbox/boards/adp_xc7k_ae350.overlay

```diff
@@ -16,4 +16,10 @@
 		compatible = "mmio-sram";
 		reg = <0x00000000 0x10000000 >;
 	};
+
+	mbox-consumer {
+		compatible = "vnd,mbox-consumer";
```

Comment:
Needs converting to hwmv2 (check if any board names have changed and apply change)